src/renderer/store/modules/ytdl.js

```diff
@@ -288,10 +288,15 @@ const actions = {
             break
         }
       }
-      const locale = settings.currentLocale.replace('-', '_')
+      let locale = settings.currentLocale.replace('_', '-')
```

Comment:
Maybe include
import i18n from '../../i18n/index'
at the top
and then get the current locale with:
i18n.locale